variables.tf

```diff
@@ -51,6 +51,22 @@ variable "platform_role_override_name" {
   default     = ""
 }
 
+################################################################################
+# IAM user
+################################################################################
```

Comment:
Add two variables similar to `platform_user_override_enabled` as a `bool` and then `platform_user_override_name` as a string